code/modules/antagonists/revolution/revolution.dm

```diff
@@ -383,13 +383,28 @@
 	SSshuttle.clearHostileEnvironment(src)
 	save_members()
 
+	var/charter_given = FALSE
+
 	// Remove everyone as a revolutionary
 	for (var/_rev_mind in members)
 		var/datum/mind/rev_mind = _rev_mind
 		if (rev_mind.has_antag_datum(/datum/antagonist/rev))
 			var/datum/antagonist/rev/rev_antag = rev_mind.has_antag_datum(/datum/antagonist/rev)
 			rev_antag.remove_revolutionary(FALSE, . == STATION_VICTORY ? DECONVERTER_STATION_WIN : DECONVERTER_REVS_WIN)
-			LAZYADD(rev_mind.special_statuses, "<span class='bad'>Former [(rev_mind in ex_headrevs) ? "head revolutionary" : "revolutionary"]</span>")
+			if(!(rev_mind in ex_headrevs))
+				LAZYADD(rev_mind.special_statuses, "<span class='bad'>Former revolutionary</span>")
+			else
+				LAZYADD(rev_mind.special_statuses, "<span class='bad'>Former head revolutionary</span>")
+				if(!charter_given && rev_mind.current && rev_mind.current.stat == CONSCIOUS)
+					charter_given = TRUE
+					var/obj/structure/closet/supplypod/bluespacepod/syndicate/pod = new()
+					pod.explosionSize = list(0,0,0,0)
+					new /obj/item/station_charter/revolution(pod)
+					var/turf/landing_location = get_turf(rev_mind.current)
+					new /obj/effect/pod_landingzone(landing_location, pod)
```

Comment:
We really need a separate proc for this, feels like we're doing this a lot. I don't know if it's a requirement here, but would be nice to have.